src/hooks/use-message-actions.tsx

```diff
@@ -99,7 +101,7 @@ export const useMessageActions = (message: MailMessage): Array<any> => {
 			setMsgRead([message.id], message.read, t, dispatch, folderId, replaceHistory, deselectAll)
 		);
 		arr.push(moveMessageToFolder([message.id], t, dispatch, false, createModal, deselectAll));
-		arr.push(printMsg(message.id, t, timezone));
+		arr.push(printMsg(message.id, t, message, dispatch, account));
```

Comment:
instead of passing message.id & message separately just pass message and wherever you are using the first parameter replace it with message.id there

Reply:
also please remove timezone if it's not required anymore